hooks/pre/30-upgrade.rb

```diff
@@ -1,17 +1,23 @@
+def start_service(service)
+  status = Kafo::Helpers.execute("service-wait #{service} status")
```

Comment:
Guessing katello-service doesn't have a start --only type of option such that we aren't relying on two service interfaces?